### PR TITLE
Database.js

### PR DESCRIPTION
**Pull request by swathij943**

### CODE REVIEW

**Review by sthan3990 — APPROVED** [review]

File: db/seeds/01_users.sql
Comment (sthan3990) [inline db/seeds/01_users.sql:1]: Password1 and Password2 needs to be hashed I think .....

File: routes/login.js
Comment (sthan3990) [inline routes/login.js:1]: res.redirect('/**itema**') - Is this a typo?

File: views/users.ejs
Comment (sthan3990) [inline views/users.ejs:1]: <script src="/vendor/jquery-3.0.0.js"></script> 
^^ why are we hosting this ourselves?

Why not just use  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>

**Review by sthan3990 — CHANGES_REQUESTED** [review]

File: routes/login.js
Comment (swathij943) [reply routes/login.js:1]: yeah

File: views/users.ejs
Comment (swathij943) [reply views/users.ejs:1]: we can use it, will change it

**Review by sthan3990 — APPROVED** [review]

**Review by sthan3990 — APPROVED** [review]

**Pull request merged**